app/styles/components/donations/create-donation.scss

```diff
@@ -0,0 +1,59 @@
+.create-donation {
+  display: flex;
+  flex-direction: column;
+
+  @include span-columns(5);
+  @include shift(7/2);
+
+  @include media($lg-screen) {
+    @include span-columns(8);
+    @include shift(2);
+  }
+
+  @include media($md-screen) {
+    @include shift(1);
+    @include span-columns(10);
+  }
+
+  @include media($sm-screen) {
+    @include shift(0);
+    @include span-columns(12);
+  }
```

Comment:
There's a lot of shared code here between the two styles of donation pages, this and the checkout flow.